src/components/frame/v5/pages/ColonyPreviewPage/ColonyPreviewPage.tsx

```diff
@@ -153,83 +150,126 @@ const ColonyPreviewPage = () => {
+                {
+                  hidden: inviteIsValid && !wallet,
+                },
+              )}
+            >
+              {formatText(MSG.info, {
+                noWallet: !wallet && inviteIsInvalid,
+              })}
+            </p>
+            <div className="pt-9 md:pt-8">
+              {inviteCode ? (
+                <InfoBanner
+                  icon={inviteIsValid ? Confetti : Password}
+                  title={formatText(MSG.infoBannerTitle, {
+                    needsToRequestAccess: inviteIsInvalid,
+                  })}
+                  text={formatText(MSG.infoBannerDescription, {
+                    needsToRequestAccess: inviteIsInvalid,
+                    colony: (
+                      <span className="font-bold">{colonyDisplayName}</span>
+                    ),
+                  })}
+                  variant={inviteIsValid ? 'success' : 'error'}
+                />
+              ) : (
+                <CardWithCallout
+                  className="border-grey-200"
+                  title={
+                    <div className="flex w-full flex-wrap items-center gap-2 sm:flex-nowrap sm:gap-4">
+                      <ColonyAvatar
+                        colonyImageSrc={
+                          colonyMetadata?.thumbnail ||
+                          colonyMetadata?.avatar ||
+                          undefined
+                        }
+                        colonyAddress={colonyAddress || ADDRESS_ZERO}
+                        colonyName={colonyDisplayName}
+                        size={24}
+                      />
+                      <h1 className="inline text-md font-medium">
+                        {colonyDisplayName}
+                      </h1>
+                    </div>
+                  }
+                >
+                  {formatText(MSG.restrictedAccessMessage)}
+
+                  <SocialLinks
+                    className="mt-3.5 w-full sm:ml-auto sm:w-auto"
+                    externalLinks={socialLinks}
+                  />
+                </CardWithCallout>
+              )}
+            </div>
+          </div>
+          <div className="hidden w-full md:block">
+            {!wallet ? (
+              <Button isFullSize onClick={connectWallet}>
+                {formatText(MSG.connectWalletButton)}
+              </Button>
+            ) : (
+              <>
+                {inviteIsValid && (
+                  <Button isFullSize onClick={validateInviteCode}>
+                    {formatText(MSG.joinColonyButton)}
+                  </Button>
+                )}
+                {inviteIsInvalid && (
+                  <a href={REQUEST_ACCESS} target="_blank" rel="noreferrer">
+                    <Button isFullSize>
+                      {formatText(MSG.requestAccessButton)}
+                    </Button>
+                  </a>
+                )}
+              </>
+            )}
           </div>
```

Comment:
And then we can also use it here 
`<div className="hidden w-full md:block">{actions}</div>`